### ISSUE TITLE
[New build - DEFAULT] 20w18f: Hovercraft physics

### ISSUE THREAD

**Issue by tsunamayo** [tsunamayo/Starship-EVO#1945]
Hovercraft repulsor model have been improved, Hovercraft will now be much harder to flip.

Change and Features:
 - Hovercraft physics improved 

Player Suggestions:
#1939 Children entity Yoke warning message

Bug Fixes:
#1933 1924 Rename Entity button
#1931 Export Entity button
#1879 Hovercraft too bouncy
#1934 Entity tab widget
#1936 Add server menu exit
#1937 No Entity Menu on paint / no item selected
#1940 Landing pad incorrectly freed on landing
#1938 Entity Preview on children update

**Comment by Sullos** [comment]
Hey Tsuna, just letting you know the build never reached us. Thanks

**Comment by tsunamayo** [comment]
Okay sorry I messed-up, is it okay now? Thanks you are on the ball mate!

**Comment by EndoSkull** [comment]
My export button doesn't seem to "push" is there a requirement before it works? And if it does work where does it export to (file path?)

**Comment by ExodistSKY1** [comment]
@EndoSkull I am, also having this issue, the file path it extracts to is:

%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\Save_Data

Hope this helps.

**Comment by Majikmonster** [comment]
I'm having the same problem. I can name an entity, but the export function doesn't work at all. No saving ships in the last 2 builds :(

**Comment by tsunamayo** [comment]
@Majikmonster @ExodistSKY1  yes there is an issue with export, please wait for the fix, coming live soon. Thanks

**Comment by tsunamayo** [comment]
New fix is live!
